ui/src/controllers/renderApp.js

```diff
@@ -23,7 +23,8 @@ export default async (req, res, next) => {
 
     data.state = {
       app: {
-        RESTRICT_CREATE_ORGANISATION: boolean(defaultTo(process.env.RESTRICT_CREATE_ORGANISATION, true))
+        RESTRICT_CREATE_ORGANISATION: boolean(defaultTo(process.env.RESTRICT_CREATE_ORGANISATION, true)),
+        ENABLE_SINGLE_STATEMENT_DELETION: boolean(defaultTo(process.env.ENABLE_SINGLE_STATEMENT_DELETION, true))
```

Comment:
Shouldn't there be something in here for batch deletions too @cbishopvelti?

Reply:
One scope for deletion statements, however, this wouldn't be visible unless single statement deletion is enabled.

Reply:
OK, happy for there to only be one scope, but in which case we should rename the env variable accordingly and also remove the additional env that _does_ reference batch deletions!

https://github.com/LearningLocker/learninglocker/pull/1346/files/df8441e126271e968fe32932a78411d9d9d4ac52#diff-cbfd64a28982a1818f2b5f16e7f9d634R218

Reply:
Ok thanks James 👍